redis.conf

```diff
@@ -1623,6 +1623,7 @@ latency-monitor-threshold 0
 #  x     Expired events (events generated every time a key expires)
 #  e     Evicted events (events generated when a key is evicted for maxmemory)
 #  t     Stream commands
+#  d     Module commands
```

Comment:
do we wanna mention "Module commands" or "Module types"?
normally the only one's touching string types are stream commands, etc.
but are we talking here of keyspace notifications that are coming from module commands or on behalf of module type keys?

it's true that the module can do whatever it wants with this, but if we document this and let clients use it, we must give some guidelines as to it's purpose (unlike list and streams, it's a bit vague without knowing how it's used by the modules)